uo_sys_framework/CaptureZoneModule/cfgFunctions/common/fn_Loop.sqf

```diff
@@ -70,9 +70,9 @@ GVAR(DOUBLES(PFHhandle,_logic)) = [{
                 if !(_silent) then {
                     if (_automessages) then {
                         private _msg = format ["%1 is uncontested!",_zoneName];
-                        _msg remoteExec ["hint"];
```

Comment:
That must have been fun haha! I am still surprised by anyone not muting their system sounds in ArmA...

Reply:
it was incessant. tortuous and embarrassing. Really fun time.